ql/src/java/org/apache/hadoop/hive/ql/ddl/table/lock/LockTableOperation.java

```diff
@@ -34,7 +34,7 @@ public LockTableOperation(DDLOperationContext context, LockTableDesc desc) {
 
   @Override
   public int execute() throws HiveException {
-    Context ctx = context.getDriverContext().getCtx();
+    Context ctx = context.getDriverQueue().getCtx();
```

Comment:
apparently this is not a queue: why would a queue would have `getCtx()` method

Reply:
fixed.